python/cuml/test/dask/test_kneighbors_classifier.py

```diff
@@ -79,25 +79,22 @@ def dataset(request):
 def exact_match(output1, output2):
     l1, i1, d1 = output1
     l2, i2, d2 = output2
-    l2 = l2.squeeze()
+    i1, i2 = i1.squeeze(), i2.squeeze()
+    d1, d2 = d1.squeeze(), d2.squeeze()
 
     # Check shapes
     assert l1.shape == l2.shape
     assert i1.shape == i2.shape
     assert d1.shape == d2.shape
 
     # Distances should match
-    d1 = np.round(d1, 4)
-    d2 = np.round(d2, 4)
-    assert np.mean(d1 == d2) > 0.98
+    assert np.array_equal(d1, d2)
```

Comment:
Since the KNeighborsClass/Reg are just giving predictions, we shouldn't need to compare the indices and distances right? Won't the NearestNeighbors test catch them?